src/messenger-shortcuts.js

```diff
@@ -127,7 +155,21 @@ function openHelp() {
   mute();
   setTimeout(function() {
     document.querySelector('div[role="dialog"] h2').innerHTML = "Keyboard Shortcuts for Messenger";
-    document.querySelector('div[role="dialog"] h2~div').innerHTML = HELP_HTML;
-    document.querySelector('div[role="dialog"] h2~div~div').remove();
-  }, 100);
+      document.querySelector('div[role="dialog"] h2~div').innerHTML = HELP_HTML;
```

Comment:
Why the indents here? In general, can you fix your indentation to match the original code (i.e., two spaces)?